btcwallet/btcwallet_test.go

```diff
@@ -0,0 +1,123 @@
+package btcwallet
+
+import (
+	"sync"
+	"testing"
+)
+
+func assertNoError(t *testing.T, err error) {
+	t.Helper()
+	if err != nil {
+		t.Errorf("No error expected but got: %v", err)
+	}
```

Comment:
This will always print `No error expected but got: nil` 😃

Reply:
> This will always print `No error expected but got: nil`

*Stranger Things*

Reply:
My mistake